db/models/topics_models.js

```diff
@@ -0,0 +1,18 @@
+const db=require('../connection.js')
+
+const selectAllTopics=()=>{
+
+    return db.query(`
+    SELECT * FROM topics;`)
+    .then(({rows})=>{
+        if(rows.length===0){
+            return Promise.reject({status:404,msg:"not found"})
```

Comment:
Is it an error if there are no topics in this database? Ideally of course there should be some. But a select all with no where clause wouldn't really error if there were no rows.